### ISSUE TITLE
DFGT wheel jerkiness with FFB

### ISSUE THREAD

**Issue by jackun** [jackun/USBqemu-wheel#3]
http://forums.pcsx2.net/Thread-Qemu-USB-Wheel-Storage-mod?pid=358423#pid358423
http://forums.pcsx2.net/Thread-Qemu-USB-Wheel-Storage-mod?pid=355645#pid355645

**Comment by jackun** [comment]
Seems by design

**Comment by JamesLewisLiu** [comment]
What's the right PCSX2 and plugin version should I use?My DFGT cannot work properly with this.
I tried 1.3.0,1.4.0,copules of 1.5.0 git version on PCSX2 and 0.8.0 0.8.1 0.8.2 for plugin but all for them report "rawinput: Could not enqueue report data: 15" and the wheel itself is not in the center and basically not working when launch the game and buttons all mapped wrong.

**Comment by jackun** [comment]
Uncheck "Logitech wheel pass-through".

Do PS2 games support DFGT in its native mode? Plugin itself doesn't change its mode so the HID report format is probably wrong hence messed up inputs. It could work in GTF/DF/DFPro mode probably but isn't necessary.

Enqueue error is because game doesn't read input data as fast as wheel produces it if using pass-through. Might cause stale/laggy input.

**Comment by JamesLewisLiu** [comment]
> Uncheck "Logitech wheel pass-through". (Deemit, it defaults to enabled.)
> 
> Do PS2 games support DFGT in its native mode? Plugin itself doesn't change its mode so the HID report format is probably wrong hence messed up inputs. It could work in GTF/DF/DFPro mode probably but isn't necessary.
> 
> Enqueue error is because game doesn't read input data as fast as wheel produces it if using pass-through. Might cause stale/laggy input.

After uncheck the pass-through it works flawlessly,thank you!
I mainly play GT4 so it should support DFGT?Or not?

**Comment by jackun** [comment]
IIRC Driving Force Pro is the newest it supports, DFGT launched with GT5 on PS3.

**Comment by JamesLewisLiu** [comment]
> IIRC Driving Force Pro is the newest it supports, DFGT launched with GT5 on PS3. 900 degree range is supported anyway so no difference.

I think I saw some videos on YouTube that someguy did connect it to PS2...That's weird

**Comment by jackun** [comment]
Yeah, newer wheels start up as "Driving Force" initially and DFGT, G25, G27 support acting as old wheels. On PC, driver then sends a command that switches it into "native" mode with full supported features.

For PCSX2, there's no difference what wheel you use if you can set it to at least 270/900 degrees, mostly :P
